config/default/pyproject.toml.j2

```diff
@@ -44,13 +44,12 @@ target-version = ["py38"]
 [tool.codespell]
 ignore-words-list = "%(codespell_ignores)s"
 {% endif %}
-{% if dependencies_ignores or dependencies_mapping %}
 
 [tool.dependencychecker]
+'plone.base' = ['AccessControl', 'Acquisition', 'ZODB', 'Products.BTreeFolder2', 'Products.CMFCore', 'Products.CMFDynamicViewFTI', 'transaction', 'zExceptions', 'zope.component', 'zope.deprecation', 'zope.i18n', 'zope.i18nmessageid', 'zope.interface', 'zope.publisher', 'zope.schema']
```

Comment:
Looking at the `setup.cfg` of `plone.base` I would expect a few more in here:

```
    Zope
    plone.batching
    plone.registry
    plone.schema
    plone.z3cform
```

I am not sure if `Zope` should be added in this list, or if that would result in ignoring all `zope.*` packages.

Reply:
I left Zope out in purpose, as it has it's own mapping, I would say that Zope and plone.base as two dependencies are not that bad 😅

Same for the plone packages, I would expect that list to grow back and forth, so it feels not that useful to keep track of those IMHO